index.html

```diff
@@ -228,7 +228,8 @@ <h2> Multi-Post Stories</h2>
         <h2>About Myself</h2>
       </div>
       <p>
-        Hello, I'm Gilbert Mutai a software developer! I can help you build a product , feature or website.Look through some of my work
+        Hello, I'm Gilbert Mutai a software developer! I can help you build a product , feature or website.Look through
+        some of my work
```

Comment:
- [ ] Kindly make sure you remove that background image on the `card_detail section`
![image](https://user-images.githubusercontent.com/97100091/193897080-e4b43948-d344-4535-b081-f5a319add435.png)

Figma :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193897193-f7656112-441c-4636-98a3-349a74935092.png)